src/Core/Entities/ISubscriber.cs

```diff
@@ -1,15 +1,17 @@
 ﻿using Bit.Core.Enums;
 
+#nullable enable
+
 namespace Bit.Core.Entities;
 
 public interface ISubscriber
 {
     Guid Id { get; }
     GatewayType? Gateway { get; set; }
-    string GatewayCustomerId { get; set; }
-    string GatewaySubscriptionId { get; set; }
-    string BillingEmailAddress();
-    string BillingName();
+    string? GatewayCustomerId { get; set; }
+    string? GatewaySubscriptionId { get; set; }
+    string? BillingEmailAddress();
+    string? BillingName();
```

Comment:
Why do these not need to be initialized `null!`?

Reply:
Since this is just an interface, that would be an implementation albeit a super simple one and since all the implementers will for sure return a non-null I can use non-null as the lowest common denominator.